src/manual_wrappers.jl

```diff
@@ -47,6 +71,34 @@ macro gsl_function(f)
     )
 end
 
+gsl_function_f_helper(x::Cdouble, (f,))::Cdouble = f(x)
+gsl_function_df_helper(x::Cdouble, (f,df,))::Cdouble = df(x)
+gsl_function_fdf_helper(x::Cdouble, (f,df,fdf))::Tuple{Cdouble,Cdouble} = fdf(x)
```

Comment:
This is wrong. Also I suggest using different helper for fdf directly which avoid the creation of a closure unnecessarily.

See https://github.com/JuliaMath/GSL.jl/commit/8a5a1d1e89de18a941e8438b2a2f04e0a4a68ee9

Though this is currently a moot point since there's no user of this.